q3-2024.json

```diff
@@ -229,5 +229,33 @@
     "quadrant" : "trial",
     "isNew" : "TRUE",
     "description" : "https://github.com/xataio/pgroll, https://github.com/fabianlindfors/reshape"  
-  }
+  },	
+  {
+		"name": "AWS Gateway API v2 (WebSockets)",
+		"ring": "trial",
+		"quadrant": "platforms",
+		"isNew": "FALSE",
+		"description": "Amazon API Gateway v2 WebSockets allows you to create and manage WebSocket APIs, which enable bidirectional communication between clients and servers.  This will be used to replace the current RosterApps Payroll communication method between browser and server."
+	},
+	{
+		"name": "AWS SNS",
+		"ring": "trial",
+		"quadrant": "platforms",
+		"isNew": "FALSE",
+		"description": "Amazon Simple Notification Service (SNS) is a fully managed messaging service provided by AWS. It enables the delivery of messages from publishers to subscribers using a variety of endpoints, such as email, SMS, HTTP/HTTPS, and AWS Lambda functions. This will be used to replace the current RosterApps Payroll communication method between server and browser."
+	},
+	{
+		"name": "AWS SQS",
+		"ring": "assess",
+		"quadrant": "platforms",
+		"isNew": "FALSE",
+		"description": "Amazon Simple Queue Service (SQS) is a fully managed message queuing service provided by AWS. It allows you to decouple and scale microservices, distributed systems, and serverless applications by enabling communication between software components through message queues.  This will be used to replace the current RosterApps Payroll communication method between browser and server to reduce WebSocket connections."
+	},
+	{
+		"name": "AWS Fargate (ECS .NET)",
```

Comment:
@bking0 so we already have an AWS Fargate footprint but maybe we speak to the idea of a .NET Core application being assess?

Reply:
Yep, that's why i put the .NET in the name ;).  I can explicitly call out that we have existing Fargate tasks for Java but it is new to .NET.

Reply:
But I guess my point is whether .NET Core apps would be the new thing here?